Assets/_BForBoss/_Core/Scripts/Managers/SandboxWorldManager.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class SandboxWorldManager : BaseWorldManager
+    {
+        [SerializeField] private EnergySystemBehaviour _energySystemBehaviour;
+
+        protected override Vector3 SpawnLocation => _spawnLocation.position;
+        protected override Quaternion SpawnLookDirection => _spawnLocation.rotation;
+
+        protected override void Start()
+        {
+            _playerBehaviour.EnergySystem = _energySystemBehaviour;
```

Comment:
This is something I'm not quite sure about. 
Should this be in the BaseWorldManager? 
if it's in the BaseWorldManager, Then the energy system would have to be added into each and every scene.

Reply:
hmm good question, we can do this for now and put in BaseWorldManager in the future if we think that is better